test/system/030-run.bats

```diff
@@ -181,7 +181,7 @@ echo $rand        |   0 | $rand
     run_podman image exists $NONLOCAL_IMAGE
 
     # Now try running with --rmi : it should succeed, but not remove the image
-    run_podman run --rmi --rm $NONLOCAL_IMAGE /bin/true
+    run_podman 0+e run --rmi --rm $NONLOCAL_IMAGE /bin/true
     is "$output" ".*image is in use by a container" "--rmi should warn that the image was not removed"
```

Comment:
Note that this is "e", that is, the warning is actually `level=error`. Should it be?

Reply:
That LGTM.  The man page states: "After exit of the container, remove the image unless another container is using it." It seems like a good idea to log when the image cannot be removed.

Reply:
Full agreement on "there should be a warning". My question is, should it be `level=error`?

Reply:
Ah, thanks for clarifying.  I think a warning would be better than an error.  It's perfectly fine behavior which does not classify as an error IMO.